includes/payment-tokens/class-wc-stripe-ach-payment-token.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+// phpcs:disable WordPress.Files.FileName
+
+/**
+ * WooCommerce Stripe ACH Direct Debit Payment Token.
+ *
+ * Representation of a payment token for ACH.
+ *
+ * @class    WC_Payment_Token_ACH
+ * @since    9.1.1
```

Comment:
Shouldn’t we use `@since x.x.x` so it gets automatically replaced when the new version is released?

Reply:
Do we have an automated process to populate these tags, or should we leave them as x.x.x and update them manually later?

Reply:
IIRC it used to have https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1560

Reply:
Nice! updated in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/3811/commits/d0c36be603f72d6910906013929f8a752af7b705